io.github.HotShots.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.HotShots
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+
+command: hotshots
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=xdg-pictures
+  - --share=network
+  
+modules:
+- name: HotShots
+  sources:
+  - type: git
+    url: https://github.com/obiwankennedy/HotShots
```

Comment:
Should be a tagged release or a specific commit

Reply:
thank you.